cypress/support/constants.ts

```diff
@@ -0,0 +1,13 @@
+export const selectors = {
```

Comment:
Hm, mulig dette objektet burde blitt brukt som datacy ID-er til komponentene også? Slik at man slipper å ha 1 id og 1 selector som må matche.

Reply:
Tja, vet ikke om vi vil ha en direkte avhengighet mellom cypress og prodkoden? Dessuten genereres vel noen av idene dynamisk i komponentene, så usikker på hvordan vi skulle gjort det. Kan se om jeg finner ut noe om best practice for å organisere slike ider...

Reply:
Litt usikker om det er lurt selv :) Men de dynamiske id-ene burde vi kanskje unngå, ettersom at poenget med kode-uavhengige test-ider forsvinner litt da

Reply:
Ja, men har ikke så lyst til å refaktorere komponentene eller legge til props bare for å støtte testene heller. Da synes jeg det er bedre å ha test-idene "duplisert" i cypress-koden. Så har vi i hvert fall ett sted i cypress man evt må oppdatere om de skulle endre seg og testene feiler.

Reply:
Jepp, høres lurt ut